tests/ci_build/Dockerfile.gpu

```diff
@@ -19,7 +19,7 @@ ENV PATH=/opt/python/bin:$PATH
 # Create new Conda environment with cuDF, Dask, and cuPy
 RUN \
     conda create -n gpu_test -c rapidsai-nightly -c rapidsai -c nvidia -c conda-forge -c defaults \
-        python=3.7 cudf=21.06* rmm=21.06* cudatoolkit=$CUDA_VERSION_ARG dask=2021.05.0 dask-cuda dask-cudf cupy \
+        python=3.7 cudf=21.06* rmm=21.06* cudatoolkit=$CUDA_VERSION_ARG dask>=2021.6.0 dask-cuda dask-cudf cupy \
```

Comment:
We are currently using the stable 21.06 version of Dask-CUDA, which has a hard dependency on the Dask version 2021.5.1

https://github.com/rapidsai/dask-cuda/blob/ef78a82ac5af8379e17857ae44cb529e3040f1eb/conda/recipes/dask-cuda/meta.yaml#L30

Upgrading to 21.08 nightly version of RAPIDS suite will pick up the latest Dask.

Reply:
Thanks for debugging it.